src/components/forms/FieldView.tsx

```diff
@@ -16,7 +17,7 @@ const FieldView = <FormValues extends FieldValues>({ register, field, formState,
     <input {...register(`answers.${index}.field.id` as Path<FormValues>)} type='hidden' value={field.id} />
     {field.type === FormFieldType.TEXT_ANSWER ? (
       <>
-        {`${field.title} ${field.required ? '*' : ''}`}
+        <Typography style={{ color: disabled ? 'grey' : 'theme.palette.text.secondary' }}>{`${field.title} ${field.required ? '*' : ''}`}</Typography>
```

Comment:
For å bruke `theme` må du bruke `sx`-prop'en og funksjon:
```suggestion
        <Typography sx={{ color: (theme) => theme.palette.text[disabled ? 'disabled' : 'primary'] }}>{`${field.title} ${field.required ? '*' : ''}`}</Typography>
```